src/components/domain/landing/LandingTabs.tsx

```diff
@@ -0,0 +1,60 @@
+'use client';
+
+import { cn } from '@/lib/utils';
+import { useEffect, useState } from 'react';
+
+const tabs = [
+  { label: '팀원평가', id: 'survey' },
+  { label: '분석지표', id: 'chart' },
+  { label: 'PR하기', id: 'profile' },
+  { label: '프로젝트 등록', id: 'register' },
+  { label: '프로젝트 검색', id: 'search' },
+];
+
+export default function LandingTabs() {
+  const [isSticky, setIsSticky] = useState(false);
+
+  useEffect(() => {
+    const handleScroll = () => {
+      setIsSticky(window.scrollY > 0);
+    };
+
+    window.addEventListener('scroll', handleScroll);
+    return () => window.removeEventListener('scroll', handleScroll);
+  }, []);
+
+  const handleClick = (id: string) => {
+    const element = document.getElementById(id);
+    if (element) {
+      const elementRect = element.getBoundingClientRect();
+      const elementTop = elementRect.top + window.scrollY;
+      const elementHeight = elementRect.height;
+      const viewportHeight = window.innerHeight;
+      const offset = elementTop - (viewportHeight / 2 - elementHeight / 2);
+
+      window.scrollTo({
+        top: offset,
+        behavior: 'smooth',
+      });
+    }
+  };
+
+  return (
+    <div
+      className={cn(
+        'w-full bg-[#1E1B4BE5]',
+        isSticky ? 'fixed left-0 right-0 top-0 z-50' : 'relative',
```

Comment:
```suggestion
        isSticky ? 'fixed left-0 right-0 top-0 z-50' : '',
```
추가로 relative는 필요 없어보입니다!

Reply:
위의 방향으로 수정했습니다. 
✅ [74de2f3](https://github.com/PRism-Works/PRism-FE/pull/144/commits/74de2f306a35f098965d791d07c3b543aa76ac25)